src/reseeding.rs

```diff
@@ -47,6 +47,7 @@ impl<R: Rng, Rsdr: Reseeder<R>> ReseedingRng<R, Rsdr> {
     /// generated exceed the threshold.
     pub fn reseed_if_necessary(&mut self) {
         if self.bytes_generated >= self.generation_threshold {
+            trace!("Reseeding RNG");
```

Comment:
Maybe change this to something like `"{} generated {} MiB, attempting to reseed"`?

Reply:
I could change it to "Reseeding RNG after {} bytes", but I don't think I can do much better than that. Not sure it's really useful changing units?